### PR TITLE
DOC-1844 K8s: Edit RERC 

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Jira: DOC-1844
Readme source material: https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/active_active_database_readme.md#update-existing-participating-cluster-rerc-details
Staged preview: https://docs.redis.com/staging/DOC-1844/kubernetes/active-active/preview/edit-rerc/

### CODE REVIEW

**Comment by kaitlynmichael** [comment]
Not ready for review: waiting to meet with SME to get clarification/feedback

File: content/kubernetes/active-active/preview/edit-rerc.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  The output should look like this:
+
+  ```sh
+   NAME        STATUS   SPEC STATUS   LOCAL
+    rerc1   Active   Valid         true
+  ```
+
+  To troubleshoot invalid configurations, view the RERC custom resource events and the [Redis Enterprise operator logs]({{<relref "/kubernetes/logs/">}}).
+
+1. Verify the status each REAADB using that RERC is "Active" and the spec status is "Valid."
```
Comment (rrelledge) [inline content/kubernetes/active-active/preview/edit-rerc.md:72]: Missing word?
```suggestion
1. Verify the status of each REAADB using that RERC is "Active" and the spec status is "Valid."
```

File: content/kubernetes/active-active/preview/edit-rerc.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Use the `kubectl patch rerc <rerc-name> --type merge --patch` command to patch the local RERC custom resource with your changes. For a full list of available fields, see the [RERC API reference](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_remote_cluster_api.md).
+
+The following example edits the `dbFqdnSuffix` field for the RERC named `rerc1`.
+
+```sh
+kubectl patch rerc rerc1 --type merge --patch \
+'{"spec":{"dbFqdnSuffix": "-example2-cluster-rec1-ns1.redis.com"}}'
+```
+
+## Update RERC secret
```
Comment (rrelledge) [inline content/kubernetes/active-active/preview/edit-rerc.md:32]: The numbered steps in this section are not being rendered correctly. Adding tabs/spaces before the non-numbered sentences and code snippets should fix this issue.

**Review by rrelledge — CHANGES_REQUESTED** [review]

**Review by alon-zada — APPROVED** [review]
Great!

**Pull request merged**